src/styles/variables.scss

```diff
@@ -0,0 +1,5 @@
+$white: #FFF;
+$blue-accent: #00ACDC;
+$secondary: #616070;
+$main-accent: #060B35;
+$cards-column-count: 1;
```

Comment:
Now some values are duplicated, need to fix it
<img width="419" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/e1dece19-b849-488d-ab52-99bc2245aa39">